app/pages/project/field-guide-container.spec.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import FieldGuideContainer from './field-guide-container';
+
+const guide = {
+  items: [{
+    content: "Facts about this animal",
```

Comment:
Linter is warning about double quotes. You've used single quotes everywhere else, if IIRC.